styles/access-layer.scss

```diff
@@ -25,6 +25,28 @@ body{
     line-height: 1.75;
   }
 
+  &__exchanges-logos{
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    justify-content: center;
+
+    & a:nth-child(4){
```

Comment:
This will bite you when we add more exchanges. If we remove some and add one that doesn't need it it will still apply the rule. Use a modifier like `--is-latoken` instead

Reply:
Agreed. Updated!